### DISCUSSION TITLE
PP-OCRv3 and v4 issues

### DISCUSSION THREAD

**Discussion by Acheerfulloser** [PaddlePaddle/PaddleOCR#12392] [Q&A]
paddleocr 2.7版本中更新了PP-OCRv4，在数据优化方面有一点问题想咨询一下：

1. PP-OCRv3的数据量和v4的数据量分别是什么数量级的，两个模型版本训练的样本重复性高吗，重复占比到多少？ 
2. 用v3剔除了得分小于0.15的样本后，PaddleOCR-release2.7/doc/doc_ch/PP-OCRv4_introduction.md的评估表格中，应该是不包括这部分样本的
  1）中文场景：在PP-OCRv4_introduction.md中v3的hmean和PP-OCRv3_introduction.md中的v3的hmean指标相差较大，这是因为什么呢，测试集剔除了置信度小于0.15的样本，应该是指标更高才对，但如果有新样本引入，v3是用之前发布的模型测试的还是在新样本集上训练的模型测试；
  2）阿拉伯语系和韩语场景：前后两年发布的.md文件记录的v3版本，acc指标波动很大，这个是因为什么？

谢谢您的解答！

**Comment by tink2123** [comment]
1. v3 数据量大概在2000万，v4 在400万左右
2. 在V4开发过程中发现V3的评估集构建存在问题，因此在V4中更换了评估集，造成指标。但是进行了公平对比的。
3. 阿拉伯语有在v3版本上有两次更新，第二次精度有大幅提升。韩语没有变化。
